test/test_factory.py

```diff
@@ -19,9 +19,7 @@
 def test_lambda_lambda():
     @make_funsor
     def LambdaLambda(
-        i: Bound,
-        j: Bound,
-        x: Funsor,
+        i: Bound, j: Bound, x: Funsor
```

Comment:
i find it much easier to read `@make_funsor` functions with vertical argument lists. can we revert changes to this file?